pkg/resources/network_policy_attachment_acceptance_test.go

```diff
@@ -49,6 +50,20 @@ func TestAcc_NetworkPolicyAttachment(t *testing.T) {
 			},
 		},
 	})
+	resource.ParallelTest(t, resource.TestCase{
+		Providers:    acc.TestAccProviders(),
```

Comment:
We use now:
```
ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
```
and additionally 
```
TerraformVersionChecks: []tfversion.TerraformVersionCheck{
			tfversion.RequireAbove(tfversion.Version1_5_0),
		},
```
Please add them

Reply:
Done!